### PR TITLE
New package: bitwig-studio-3.2.6

### PR DESCRIPTION
**Pull request by ale-saglia**
Bitwig Studio is a multi-platform music-creation system for production, performance and DJing, with a focus on flexible editing tools and a super-fast workflow.

### CODE REVIEW

File: srcpkgs/bitwig-studio/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+short_desc="DAW for music production, remixing and live performance"
+archs="x86_64"
+hostmakedepends="tar xz"
+depends="xdg-utils xcb-util-wm libbsd"
+repository=nonfree
+restricted=yes
+nostrip=yes
+
+_file="bitwig-studio-${version}.deb"
+
```
Comment (PaperMountainStudio) [inline srcpkgs/bitwig-studio/template:17]: I am pretty sure you should delete this blank line and the one blank line above

**Comment by PaperMountainStudio** [comment]
and run `xlint srcpkgs/bitwig-studio/template`, that will tell you some more improvements that can be made. You will have to have xtools package installed.

**Comment by PaperMountainStudio** [comment]
Could you make all the changes one commit?

**Comment by ale-saglia** [comment]
Should be fine now, let me know if I need to change something else.
And again thanks for the patience!

File: srcpkgs/bitwig-studio/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+pkgname=bitwig-studio
+version=3.2.6
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz"
+depends="xdg-utils xcb-util-wm libbsd"
+short_desc="DAW for music production, remixing and live performance"
+maintainer="Alessandro Saglia <ale.saglia@gmail.com>"
+license="custom:bitwig"
+homepage="bitwig.com"
```
Comment (pullmoll) [inline srcpkgs/bitwig-studio/template:11]: This should be an URL, i.e. `https://bitwig.com/`

File: srcpkgs/bitwig-studio/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+restricted=yes
+repository=nonfree
+
+do_extract() {
+	mkdir -p ${DESTDIR}
+	ar x ${XBPS_SRCDISTDIR}/bitwig-studio-${version}/${_file}
+}
+
+do_install() {
+	tar xf data.tar.xz -C ${DESTDIR}
```
Comment (pullmoll) [inline srcpkgs/bitwig-studio/template:25]: You could just as well use `bsdtar` which is available w/o adding it to `hostmakedepends`.

File: srcpkgs/bitwig-studio/template [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+post_install() {
+	install -D -m644 ${DESTDIR}/opt/bitwig-studio/EULA.rtf ${DESTDIR}/usr/share/licenses/${pkgname}/LICENSE
```
Comment (pullmoll) [inline srcpkgs/bitwig-studio/template:29]: Remove this line. The next line installs the license in the right place.

File: srcpkgs/bitwig-studio/template (hunk 2 above)
Comment (ale-saglia) [reply srcpkgs/bitwig-studio/template]: Fixed with last commit

File: srcpkgs/bitwig-studio/template (hunk 3 above)
Comment (ale-saglia) [reply srcpkgs/bitwig-studio/template]: Fixed with last commit

File: srcpkgs/bitwig-studio/template (hunk 4 above)
Comment (ale-saglia) [reply srcpkgs/bitwig-studio/template]: Fixed with last commit

**Pull request merged**